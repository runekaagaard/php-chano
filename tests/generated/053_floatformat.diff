001- 13.1031
001+ Fatal error: Call to undefined method DtlIter::floatformat() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/053_floatformat.php on line 4