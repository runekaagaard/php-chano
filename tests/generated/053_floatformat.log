
---- EXPECTED OUTPUT
13.1031
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::floatformat() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/053_floatformat.php on line 4
---- FAILED
