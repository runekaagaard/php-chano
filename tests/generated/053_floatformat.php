<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 13.1031)));
foreach ($items as $i) echo $i->input->floatformat('bar');
