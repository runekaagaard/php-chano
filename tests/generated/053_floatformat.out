Fatal error: Call to undefined method DtlIter::floatformat() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/053_floatformat.php on line 4